Advanced_Usage/Beta-program.md

```diff
@@ -15,29 +15,24 @@ This feedback can be used to help improve the final version of the feature being
 
 The goal of this program is to test those new features as well as experimenting with improvements on existing ones.
 
-By joining the OctoBot Beta Tester, you will have:
-- Early access to new features and improvements on [OctoBot](https://www.octobot.online/) and [OctoBot Cloud](https://www.octobot.cloud/)
-- A direct channel to discuss with the OctoBot Team and share your ideas on how to improve new functionalities before their release
+By joining the OctoBot Beta Tester, you will have early access to new features and improvements on [OctoBot cloud](https://app.octobot.cloud/)
```

Comment:
```suggestion
By joining the OctoBot Beta Tester, you will have early access to new features and improvements on [OctoBot cloud](https://beta.octobot.cloud/)
```

Reply:
here we are talking about improvements on octobot cloud in general

Reply:
I think this sentence is not very clear. It can be read like "By joining beta..., you will have access to new features...  on app.octobot.cloud". I think it's easy to understand that you have to go to this site to access the new features

Reply:
up